src/arr/compiler/anf-loop-compiler.arr

```diff
@@ -101,6 +101,8 @@ j-for = J.j-for
 j-raw-code = J.j-raw-code
 make-label-sequence = J.make-label-sequence
 
+var cases-dispatches = cl-empty
```

Comment:
A totally handwaving answer (because I am not sure of the actual answer):

1. Below this line, there are `js-ids = D.make-mutable-string-dict()` and `effective-ids = D.make-mutable-string-dict()`. If this doesn't get reset, those lines wouldn't get reset as well, and we would have had problem because of that long time already.
2. Experimentally, when I compiled `test.arr` to `test.js`, other libs get bundled too and somehow it gets reset.